docs/Earn/defi/provide-liquidity-dex.md

```diff
@@ -11,18 +11,18 @@ Currently, you can trade and provide liquidity to the following DEXs:
+[**Bancor**](https://bancor.network) _Ethereum_ | [Ankr/BNT](https://app.bancor.network/pools/add-liquidity/0xf833262A4f6D3663Ea02EF85a608ca097Df5b609) | n/a | n/a | n/a | n/a | n/a | n/a | |
+[**PancakeSwap**](https://pancakeswap.finance) _Binance Smart Chain_ | [Ankr/BNB](https://pancakeswap.info/pool/0x3147f98b8f9c53acdf8f16332ead12b592a1a4ae) | [aETHc/ETH](https://pancakeswap.finance/info/pool/0x77d6ecfd0cb585c979f19c0b7fc57bc652fd444e) | n/a | n/a | n/a | n/a | n/a | |
 [**Pangolin**](https://pangolin.exchange) _Avalanche_ | n/a | n/a | n/a | [aAVAXb/AVAX](https://info.pangolin.exchange/#/pair/0xaa9a58792cbfa3de9cef36a5cf0e3608a6a106b7) | n/a | n/a | n/a | |
 [**Lydia Finance**](https://www.lydia.finance) _Avalanche_ | n/a | n/a | n/a | [aAVAXb/AVAX](https://info.lydia.finance/#/pair/0xba4486e7a6f74be11fb7159d205f876168c906aa) | n/a | n/a | n/a | |
-[**TraderJoe**](https://traderjoe.xyz) _Avalanche_ | [ANKR/AVAX](https://analytics.traderjoexyz.com/pairs/0x754a67d24fa2cc9caa9596566dd72f44c32a7afc) | n/a | n/a | [aAVAXb/AVAX](https://analytics.traderjoexyz.com/pairs/0xe1231c37562ea532ba97018336836f6d69e540e1) | n/a | n/a | n/a | |
-[**QuickSwap**](https://quickswap.exchange) _Polygon_ | [ANKR/wETH](https://info.quickswap.exchange/#/pair/0x54db9acc40fd2ce8048fc36330502eedcecb71ba) | n/a | n/a | n/a | n/a | n/a | n/a | |
-[**Uniswap v3**](https://app.uniswap.org) _Polygon_ | [ANKR/wETH](https://info.uniswap.org/#/polygon/pools/0x9f883730174e6feb52365a4bada1854346216140) | n/a | n/a | n/a | n/a | n/a | n/a | |
-[**SpookySwap**](https://spookyswap.finance) _Fantom_ | [ANKR/FTM](https://info.spookyswap.finance/pair/0x313439265b03edb74265a2924a8abbdbef8726f4) | n/a | n/a | n/a | n/a | n/a | n/a | |
+[**TraderJoe**](https://traderjoe.xyz) _Avalanche_ | [Ankr/AVAX](https://analytics.traderjoexyz.com/pairs/0x754a67d24fa2cc9caa9596566dd72f44c32a7afc) | n/a | n/a | [aAVAXb/AVAX](https://analytics.traderjoexyz.com/pairs/0xe1231c37562ea532ba97018336836f6d69e540e1) | n/a | n/a | n/a | |
+[**QuickSwap**](https://quickswap.exchange) _Polygon_ | [Ankr/wETH](https://info.quickswap.exchange/#/pair/0x54db9acc40fd2ce8048fc36330502eedcecb71ba) | n/a | n/a | n/a | n/a | n/a | n/a | |
+[**Uniswap v3**](https://app.uniswap.org) _Polygon_ | [Ankr/wETH](https://info.uniswap.org/#/polygon/pools/0x9f883730174e6feb52365a4bada1854346216140) | n/a | n/a | n/a | n/a | n/a | n/a | |
+[**SpookySwap**](https://spookyswap.finance) _Fantom_ | [Ankr/FTM](https://info.spookyswap.finance/pair/0x313439265b03edb74265a2924a8abbdbef8726f4) | n/a | n/a | n/a | n/a | n/a | n/a | |
 [**Curve Finance**](https://ftm.curve.fi) _Fantom_ | n/a | n/a | n/a | n/a | n/a | [aFTMb/wFTM](https://ftm.curve.fi/factory/76) | n/a | |
 [**Ellipsis Finance**](https://ellipsis.finance) _BNB Chain_| n/a | n/a | n/a | n/a | n/a | n/a | [aBNBb/BNB](https://ellipsis.finance/pool/6) |
 
```

Comment:
These were correct before.
Ankr - The company
ANKR - the token

Reply:
reverted